src/main/java/uk/gov/hmcts/darts/usermanagement/service/UserManagementService.java

```diff
@@ -16,4 +16,5 @@ public interface UserManagementService {
 
     List<UserWithIdAndTimestamps> search(UserSearch userSearch);
 
+    List<UserWithIdAndTimestamps> getUsers(int courtCaseId, String emailAddress);
```

Comment:
This should be named `courthouseId` and potentially use `Integer` as the type?

Reply:
Corrected

Reply:
Did you change the type to `Integer`? As it actually doesn't work as `int` due to the casting of an empty `courthouse_id`